scio-google-cloud-platform/src/main/scala/com/spotify/scio/bigquery/dynamic/syntax/SCollectionSyntax.scala

```diff
@@ -100,6 +100,7 @@ final class DynamicBigQueryOps[T](private val self: SCollection[T]) extends AnyV
   }
 }
 
+@deprecated("use SCollectionTableRowOps instead")
```

Comment:
I think we can add the `@deprecated` to the method itself and point to the new method. Also we should add `since = "0.10.0"`